packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -101,7 +108,11 @@ export const ToggleSwitch = React.forwardRef((props, ref) => {
         </Box>
       )}
       {label && (
-        <Box as='span' className='iui-toggle-switch-label'>
+        <Box
+          as='span'
+          className={cx('iui-toggle-switch-label', labelClassName)}
+          {...restLabelProps}
```

Comment:
Instead of creating `restLabelProps` like this, you can reorder this and use `labelProps` directly.

```suggestion
          {...labelProps}
          className={cx('iui-toggle-switch-label', labelProps?.className)}
```

Reply:
adressed this in my last push

Reply:
That's not the same thing. You are still destructuring `className` from `labelProps`, which is unnecessary because you can just use `labelProps.className`.

Reply:
gotcha. missed that part. fixed it